debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnection.java

```diff
@@ -468,9 +467,7 @@ public String connectionString() {
 
     public static class MySqlConnectionConfiguration {
 
-        protected static final String JDBC_PROPERTY_LEGACY_DATETIME = "useLegacyDatetimeCode";
```

Comment:
@ani-sha I've commented on this one yesterday but the comment probably got lost :-(.
This and the following property are related to JDBC driver not our legacy implementation. Could you please create a separate Jira for this change and extract into a separate commit just in case we would need to revert it for some reason? Thanks

Reply:
@jpechane I should add the props back & its related change with a separate commit & Jira task?